module/Application/view/layout/layout.phtml

```diff
@@ -13,17 +13,15 @@
                 'type' => 'image/vnd.microsoft.icon',
                 'href' => $this->basePath() . '/img/favicon.ico'
             ])
-            ->appendStylesheet($this->basePath() . '/bower_components/mediaelement/build/mediaelementplayer.min.css')
-            ->appendStylesheet($this->basePath() . '/css/font.css')
-            ->appendStylesheet($this->basePath() . '/css/screen.css') ?>
+            ->appendStylesheet($this->basePath() . '/' . $this->asset('css/styles.css'))
+            ->appendStylesheet('https://cdn.jsdelivr.net/npm/font-awesome@4.7.0/css/font-awesome.min.css')
```

Comment:
Why is this one via a CDN, and not part of the styles we serve? If we serve it, no network connection is required when testing...

Reply:
The same we are doing in zf3-web: https://github.com/zendframework/zf3-web/blob/master/templates/layout/default.phtml#L11

I agree that on testing it requires network connection, but I don't think so it's a big issue. When we compile it into our assets then we will serve much bigger css file. I think external library like FA should be server from their CDN, and it could be already cached in many users browser.